usaspending_api/recipient/v2/views/recipients.py

```diff
@@ -392,6 +394,20 @@ def get(self, request, recipient_id):
         return Response(result)
 
 
+@method_decorator(deprecated, name="get")
+class RecipientOverViewDuns(RecipientOverView):
+    """
+    <em>Deprecated: Please see <a href="../recipient/99a44eeb-23ef-e7c4-1f84-9a695b6f5d2e-R">this endpoint</a> instead.</em>
```

Comment:
Two two endpoints, old and new, are:
* /api/v2/recipient/duns/99a44eeb-23ef-e7c4-1f84-9a695b6f5d2e-R/
* /api/v2/recipient/99a44eeb-23ef-e7c4-1f84-9a695b6f5d2e-R/

In order for this href to work it would need to be `../../99a44eeb-23ef-e7c4-1f84-9a695b6f5d2e-R`. The current changes result in the following link: `/api/v2/recipient/duns/recipient/99a44eeb-23ef-e7c4-1f84-9a695b6f5d2e-R/`